app/steps/ui/deceased/alias/template.html

```diff
@@ -4,8 +4,8 @@
 {% from "widgets/radiobuttons.html" import optionsRadio %}
 
 {% block question %}
-    <h1 class="form-title heading-large">{{content.question}}</h1>
-    <p>{{content.paragraph1}}</p>
+    <h1 class="form-title heading-large">{{content.question | replace('{deceasedName}', fields.deceasedFullName.value) | safe }}</h1>
```

Comment:
As `deceasedName` and `deceasedFullName` refer to the same data it would be nice to have the variables names the same, can we change one of these so they are both the same?